jme3-core/src/main/resources/Common/MatDefs/Light/PBRLighting.frag

```diff
@@ -217,11 +217,11 @@ void main(){
         Roughness = 1.0 - glossiness;
         vec3 fZero = specularColor.xyz;
     #else
-        float specular = 0.5;
-        float nonMetalSpec = 0.08 * specular;
+        float nonMetalSpec = 0.08; //nonMetalSpec is always constant value of 0.8 in metallic workflow
         vec4 specularColor = (nonMetalSpec - nonMetalSpec * Metallic) + albedo * Metallic;
         vec4 diffuseColor = albedo - albedo * Metallic;
-        vec3 fZero = vec3(specular);
+        vec3 fZero = vec3(0.04); 
+        fZero = mix(fZero, albedo.rgb, Metallic);          
```

Comment:
Another nit but couldn't you combine this in a one liner? `vec3 fZero = mix(vec3(0.04), albedo.rgb, Metallic)`?
Or is there a specific meaning?

Reply:
I think there's a few segments of code like this that could probably be condensed in the PBR shader.

It looks like the shader was originally written fairly verbosely, and I assume that was because the original author was creating this shader based on a reference doc full of PBR math and equations, so he was likely focused more on accurately following the pbr standards than creating condensed code. Which is understandable very early in the development process, since its often times easier to debug verbose code than it is to debug a big 1 liner.

But I think it should be okay to condense some of this code now that our PBR shader is in a stable state. So I'll go ahead and combine it into one line.